airbyte-integrations/connectors/destination-gcs/sample_files/configured_catalog.json

```diff
@@ -0,0 +1,27 @@
+{
+  "streams": [
+    {
+      "sync_mode": "full_refresh",
+      "destination_sync_mode": "append",
+      "stream": {
+        "name": "ab-airbyte-testing",
+        "supported_sync_modes": ["full_refresh"],
```

Comment:
Why are you adding this file?

Reply:
This file is necessary to verify the "write" command locally. I see the same file in other connector, so I included in gcs too.